### ISSUE TITLE
Missing HTML Lang Attribute

### ISSUE THREAD

**Issue by stepanfedyanov** [stepanfedyanov/fedyanov.dev#18]
The <html> element does not have a lang attribute, which may cause incorrect screen reader pronunciation.
